### DISCUSSION TITLE
part-ii/2017-68/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#394] [Question comments]
Part II 2017 Galois Theory: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2017-68/

**Comment by artie2000** [comment]
If $f=g_1\dots g_k$ is a factorisation into irreducibles of $f\in\mathbb{F}_p[X]$, then $\text{Gal}(f/\mathbb{F}_p)=C_d$, where $d=\text{lcm}(g_1,\dots,g_k)$.

I don't think this is in the course, but it's a straighforward exercise. It also makes the rest of the question much easier.

<details>
<summary>Answers:</summary>

(i) $C_2$

(ii) $C_3$

(iii) $C_2$

</details>
